<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>119</height>
    </rect>
   </property>
   <property name="title">
    <string>Relays</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="relay1_cbox">
      <property name="text">
       <string>Relay 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="relay2_cbox">
      <property name="text">
       <string>Relay 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="relay3_cbox">
      <property name="text">
       <string>Relay 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="relay4_cbox">
      <property name="text">
       <string>Relay 4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
